content/case_histories/bookpurnong/properties.rst

```diff
@@ -24,16 +24,19 @@ sections from the river bed to the bank both before and after SIS.
 .. (Figure :numref:`booky-salinity`).
 
 
-.. .. figure:: ./images/booky-drills.jpg
-..     :align: left
-..     :scale: 80%
-..     :name: booky-drills
+.. figure:: ./images/booky-drills.jpg
+ :align: left
+ :scale: 80%
+ :name: booky-drills
+
+Locations of the drills where groundwater samples were taken for the salinity analysis.
 
 .. figure:: ./images/booky-salinity.jpg
     :align: left
     :scale: 80%
     :name: booky-salinity
 
+Groundwater salinity measured by electric conductivity (EC).
```

Comment:
also, is `EC` used elsewhere in this explanation? otherwise, I would consider removing it